testing/features/support/helpers/page.rb

```diff
@@ -69,6 +70,14 @@ def height(fallback: 800)
       ENV.fetch("BROWSER_HEIGHT", fallback)
     end
 
+    def wait_for_second_window
+      Selenium::WebDriver::Wait.new.until { page.windows.length == 2 }
+    end
+
+    def second_window
+      (page.windows.to_a - [page.current_window]).first
```

Comment:
The array casting with the first object is because Capybara Collections aren't Enumerable. This is also present in some other areas of the codebase